java/ql/src/Security/CWE/CWE-330/InsecureRandomness.qhelp

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+	<overview>
+		<p>
+			Using a cryptographically weak pseudo-random number generator to generate a security-sensitive value,
+			such as a password, makes it easier for an attacker to predict the value.
+		</p>
+		<p>
+			Pseudo-random number generators generate a sequence of numbers that only approximates the properties
+			of random numbers. The sequence is not truly random because it is completely determined by a
+			relatively small set of initial values, the seed. If the random number generator is
+			cryptographically weak, then this sequence may be easily predictable through outside observations.
```

Comment:
```suggestion
			Pseudo-random number generators generate a sequence of numbers that only approximates the properties
			of random numbers. The sequence is not truly random because it is completely determined by a
			relatively small set of initial values (the seed). If the random number generator is
			cryptographically weak, then this sequence may be easily predictable through outside observations.
```

V minor